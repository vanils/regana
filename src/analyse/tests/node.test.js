
const analyseNode = require('../node.js');

describe('analyse', () => {
  describe('#node', () => {
    test('should throw error with invalid type', () => {
      expect(() => {
        analyseNode({ type: 'Unknown' });
      }).toThrowErrorMatchingSnapshot();
    });
  });
});