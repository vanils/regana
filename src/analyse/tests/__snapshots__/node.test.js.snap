// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`analyse #node should throw error with invalid type 1`] = `"Failed to parse, unknown entity type 'Unknown'"`;
